surface-model/surfacemodel_ontology.ttl

```diff
@@ -0,0 +1,17 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix gax-core: <https://w3id.org/gaia-x/core#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix surfacemodel: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/surfacemodel/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+surfacemodel: a owl:Ontology ;
+    rdfs:label "ontology definition for surfaceModel"@en ;
+    dcterms:contributor "Sebastian Tuttas (3DM)" ;
+    owl:versionInfo "0.1"^^xsd:float .
+
+surfacemodel:surfaceModel a owl:Class ;
+    rdfs:label "class definition for surfaceModel" ;
+    rdfs:comment "Attributes for surface model"@en ;
+    rdfs:subClassOf gax-core:Resource .
```

Comment:
should be gx:DataResource.
Please also add the gx prefix 
`@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .`
to both ontology and SHACL

Reply:
fixed

Reply:
still unchanged. Did you forget to push?

<img width="299" alt="image" src="https://github.com/GAIA-X4PLC-AAD/ontology-management-base/assets/70939848/a045a25b-9c0e-4c12-9bc4-91e84941356b">

Reply:
done! use DataResource